airflow/models/taskinstance.py

```diff
@@ -220,7 +220,7 @@ def clear_task_instances(
                 # outdated. We make max_tries the maximum value of its
                 # original max_tries or the last attempted try number.
                 ti.max_tries = max(ti.max_tries, ti.prev_attempted_tries)
-            ti.state = State.NONE
+            ti.state = TaskInstanceState.NONE
```

Comment:
I have had an issue with `TaskInstanceState.NONE`, it gives the string 'None' instead of a null value. We should use State.NONE instead

Reply:
Ah yeah I remember being told about this… Maybe we should remove `TaskInstanceState.NONE` altogether at some point and just use `None` (the built-in one).